### PR TITLE
Add use SPI steps for proxy startup.

### PR DESCRIPTION
**Pull request by zjcnb**

### CODE REVIEW

**Review by terrymanu — CHANGES_REQUESTED** [review]
Please remove useless space

File: docs/document/content/user-manual/shardingsphere-proxy/startup/bin.cn.md [hunk 1]
```diff
@@ -45,6 +45,9 @@ weight = 1
 当用户需要使用自定义的算法类时，可通过以下方式配置使用自定义算法，以分片为例：
 
 1. 实现 `ShardingAlgorithm` 接口定义的算法实现类。
+1. 在项目 resources 目录下创建 META-INF/services 目录。
```
Comment (terrymanu) [inline docs/document/content/user-manual/shardingsphere-proxy/startup/bin.cn.md:48]: resources and META-INF/services should around with `, just keep consist with ShardingAlgorithm

File: docs/document/content/user-manual/shardingsphere-proxy/startup/bin.cn.md [hunk 2]
```diff
@@ -45,6 +45,9 @@ weight = 1
...
+1. 在 META-INF/services 目录下新建文件 org.apache.shardingsphere.sharding.spi.ShardingAlgorithm
```
Comment (terrymanu) [inline docs/document/content/user-manual/shardingsphere-proxy/startup/bin.cn.md:49]: META-INF/services and org.apache.shardingsphere.sharding.spi.ShardingAlgorithm should around `

File: docs/document/content/user-manual/shardingsphere-proxy/startup/bin.cn.md [hunk 3]
```diff
@@ -45,6 +45,9 @@ weight = 1
...
+1. 将实现类的绝对路径写入至 文件 org.apache.shardingsphere.sharding.spi.ShardingAlgorithm
```
Comment (terrymanu) [inline docs/document/content/user-manual/shardingsphere-proxy/startup/bin.cn.md:50]: org.apache.shardingsphere.sharding.spi.ShardingAlgorithm should around `

File: docs/document/content/user-manual/shardingsphere-proxy/startup/bin.en.md [hunk 4]
```diff
@@ -45,6 +45,9 @@ please refer to [Distributed Transaction](/en/user-manual/shardingsphere-jdbc/sp
 When developer need to use user-defined algorithm, should use the way below to configure algorithm, use sharding algorithm as example. 
 
 1. Implement `ShardingAlgorithm` interface.
+1. Create META-INF/services directory in the resources directory.
+1. Create a new file org.apache.shardingsphere.sharding.spi.ShardingAlgorithm in the META-INF/services directory.
```
Comment (terrymanu) [inline docs/document/content/user-manual/shardingsphere-proxy/startup/bin.en.md:49]: resources and META-INF/services should around with , just keep consist with ShardingAlgorithm`

File: docs/document/content/user-manual/shardingsphere-proxy/startup/bin.en.md [hunk 5]
```diff
@@ -45,6 +45,9 @@ please refer to [Distributed Transaction](/en/user-manual/shardingsphere-jdbc/sp
 When developer need to use user-defined algorithm, should use the way below to configure algorithm, use sharding algorithm as example. 
 
 1. Implement `ShardingAlgorithm` interface.
+1. Create META-INF/services directory in the resources directory.
```
Comment (terrymanu) [inline docs/document/content/user-manual/shardingsphere-proxy/startup/bin.en.md:48]: resources and META-INF/services should around with `

File: docs/document/content/user-manual/shardingsphere-proxy/startup/bin.en.md [hunk 6]
```diff
@@ -45,6 +45,9 @@ please refer to [Distributed Transaction](/en/user-manual/shardingsphere-jdbc/sp
...
+1. Create a new file org.apache.shardingsphere.sharding.spi.ShardingAlgorithm in the META-INF/services directory.
+1. Absolute path of the implementation class are write to the file org.apache.shardingsphere.sharding.spi.ShardingAlgorithm
```
Comment (terrymanu) [inline docs/document/content/user-manual/shardingsphere-proxy/startup/bin.en.md:50]: org.apache.shardingsphere.sharding.spi.ShardingAlgorithm should around `

File: docs/document/content/user-manual/shardingsphere-proxy/startup/bin.cn.md (hunk 3 above)
Comment (terrymanu) [inline docs/document/content/user-manual/shardingsphere-proxy/startup/bin.cn.md:50]: Please remove useless space between `至` and `文件`

**Review by terrymanu — CHANGES_REQUESTED** [review]

File: docs/document/content/user-manual/shardingsphere-proxy/startup/bin.cn.md (hunk 3 above)
Comment (zjcnb) [reply docs/document/content/user-manual/shardingsphere-proxy/startup/bin.cn.md:50]: Ok, i will adjust it.

File: docs/document/content/user-manual/shardingsphere-proxy/startup/bin.en.md (hunk 6 above)
Comment (zjcnb) [reply docs/document/content/user-manual/shardingsphere-proxy/startup/bin.en.md:50]: > org.apache.shardingsphere.sharding.spi.ShardingAlgorithm should around `

done.

**Comment by zjcnb** [comment]
image
image

**Review by terrymanu — APPROVED** [review]

**Pull request merged**